src/ui/SubtitleEdit.csproj

```diff
@@ -2288,6 +2288,9 @@
     <None Include="Resources\color-picker-small2.png" />
     <None Include="Resources\rec32.png" />
     <None Include="Resources\Help32.png" />
+    <Content Include="DLLs\Interop.QuartzTypeLib.dll">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Just one doubts does SE requires this file at the runtime?

Because it was previously in csproj as  `<None Include="DLLs\Interop.QuartzTypeLib.dll">`
I've now added it back as 
```xml
<Content Include="DLLs\Interop.QuartzTypeLib.dll">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
</Content>
```

but if it's not required at the runtime it should be reverted back as it was original